apps/demos/Demos/Chat/Overview/jQuery/styles.css

```diff
@@ -0,0 +1,24 @@
+.demo-container {
+  padding: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.dx-chat {
+  min-width: 760px;
```

Comment:
If we set `min-width` for the Chat element, our demo will look as following
<img width="851" alt="image" src="https://github.com/user-attachments/assets/7a687a87-1f8f-492b-a87a-c8954945cf26">

Reply:
This has been fixed in the commit https://github.com/DevExpress/DevExtreme/pull/28363/commits/01a287acf4d7a1d8f2c3a171baab7ecfee47c6ed to obtain the correct reference in the jQuery approach. Please consider this solution in other frameworks

Reply:
4ffe47e04119ebbbb05117448074cca3808ff78f